packaging/tests/tas-restapi/pom.xml

```diff
@@ -47,9 +48,20 @@
     </profiles>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${commons-lang3.version}</version>
+        </dependency>
         <dependency>
             <groupId>org.alfresco.tas</groupId>
             <artifactId>restapi</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-lang3</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
What's this? Why are we declaring a dependency as direct and then excluding it from tas-restapi?

1. if we declare it as direct, it will already trump the version declared in the restapi library, as direct dependencies are closer to the root than transitive dependencies -- so it's unclear to me what purpose is served by this exclusion
2. if we wanted to simply override the version of an already existing transitive library, we could have used the `dependencyManagement` section instead of declaring it as a direct dependency

Reply:
For the 7.3.2 `alfresco-tas-restapi` hasn't been merged into community-repo https://github.com/Alfresco/alfresco-community-repo/pull/1560.
Removed unnecessary exclusion.